Content.Shared/Wieldable/Events.cs

```diff
@@ -0,0 +1,43 @@
+namespace Content.Shared.Wieldable;
+
+/// <summary>
+/// Raised directed on an item when it is wielded.
+/// </summary>
+[ByRefEvent]
+public readonly record struct ItemWieldedEvent(EntityUid User);
+
+/// <summary>
+/// Raised directed on an item that has been unwielded.
+/// Force is whether the item is being forced to be unwielded, or if the player chose to unwield it themselves.
+/// </summary>
+[ByRefEvent]
+public readonly record struct ItemUnwieldedEvent(EntityUid User, bool Force);
```

Comment:
```suggestion
/// Forced is whether the item was forced to be unwielded, rather than the player choosing to unwield it themselves.
/// </summary>
[ByRefEvent]
public readonly record struct ItemUnwieldedEvent(EntityUid User, bool Forced);
```
imo, because the unwielding has already occurred

Reply:
true